gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -402,3 +402,10 @@ liquibase:
 #          LOW: 0.2777778
 ##          MEDIUM: 6.9444445 # (25km/h)
 ##          HIGH: 69.444445 # (250km/h)
+
+# This section allows to enable implicit user consent to store remote IP and userAgent
```

Comment:
Suggestion: remove "to enable". Sot it will be
This section allows implicit user consent to store